install.sh

```diff
@@ -212,7 +220,9 @@ if [[ "${CREW_PREFIX}" != "/usr/local" ]]; then
   echo "you need to run these commands to complete your installation:"
   echo "echo 'export CREW_PREFIX=${CREW_PREFIX}' >> ~/.bashrc"
   echo "echo 'export PATH=\"\${CREW_PREFIX}/bin:\${CREW_PREFIX}/sbin:\${PATH}\"' >> ~/.bashrc"
-  echo "echo 'export LD_LIBRARY_PATH=${CREW_PREFIX}/lib${LIB_SUFFIX}' >> ~/.bashrc"
+  echo "echo 'export PATH=\"\${CREW_PREFIX}/usr/local/bin:\${CREW_PREFIX}/usr/local/sbin:\${PATH}\"' >> ~/.bashrc"
+  echo "echo 'export LD_LIBRARY_PATH=\"\${CREW_PREFIX}/lib\${LIB_SUFFIX}:\${LD_LIBRARY_PATH}\"' >> ~/.bashrc"
+  echo "echo 'export LD_LIBRARY_PATH=\"\${CREW_PREFIX}/usr/local/lib\${LIB_SUFFIX}:\${LD_LIBRARY_PATH}\"' >> ~/.bashrc"
```

Comment:
I see what you are trying to do here but I don't think it's a good idea to pull from multiple library paths because it is possible the better library may not take precedence.  Also, every time this executes, it will continue to append the same path each time and create duplicate paths.  I think a better solution is to make sure each package is installing libraries in the correct `CREW_LIB_PREFIX` location.  Finally, I believe this is only a problem on x86_64 architecture.

Reply:
I think you're right @uberhacker as I ran into library path issues in my Docker containers on x86_64 but almost never on ARM.

Reply:
is CREW_LIB_PREFIX a currently assigned environment variable?

I'm in support of this; there is a CREW_LIB_PATH assigned at `${CREW_PREFIX}/lib/crew/` yet that seems tooling specific rather than for the running environment.

**In argument:**
CREW_PREFIX should append the LD_LIBRARY_PATHS in a `crew init` function or something since CREW_PREFIX is a localized `/usr/local` for Chromebrew specifically.

Reply:
OK so CREW_LIB_PREFIX is in `crew` I had been completely using the `install.sh` for reference thus far. Hmm... yah it could be better if `install.sh` setup the environment the same way that `crew` does.

Reply:
Yeah, it was an afterthought to install Chromebrew on a relocated prefix.  It might even make sense to create a separate shell script to install on SD cards rather than trying to make it universal for any type of install.  I think you are going to run into issues with the binaries anyway since they're compiled with the `/usr/local` default prefix.

Reply:
@uberhacker I had this issue with the `crew` scripts as well.